src/Homepage/index.js

```diff
@@ -22,6 +23,23 @@ function searchResultToFormEntry(searchResult) {
   };
 }
 
+function NoOfUsersAlert() {
+  const [show, setShow] = useState(true)
+
+  return (
+    <Alert
+      key='no-of-users'
+      variant='primary'
+      show={show}
+      onClose={() => setShow(false)}
+      dismissible
+    >
```

Comment:
can we tweak the styles to avoid the margin-bottom below the alert? perhaps we can also try a slightly smaller font size + padding. looks good otherwise. will merge after copy and styling changes

Reply:
@arjun27 @asanjeevrao  resolved the comments, please check and confirm